AlfrescoApp/Model/Managers/AppConfigurationManager.m

```diff
@@ -449,11 +453,13 @@ - (void)setupConfigurationFileFromBundleIfRequiredWithCompletionBlock:(void (^)(
     AlfrescoFileManager *fileManager = [AlfrescoFileManager sharedManager];
     
     // File location to the configuration file
-    NSString *completeDestinationPath = ([AccountManager sharedManager].allAccounts.count > 0) ? [self filePathForEmbeddedConfigurationFile] : [self filePathForNoAccountsConfigurationFile];
+    BOOL areThereAccounts = ([AccountManager sharedManager].allAccounts.count > 0) || ([AccountManager sharedManager].selectedAccount != nil);
```

Comment:
As this pattern is used twice and also it's inferring information from variable state, it may have been better to have this logic inside the AccountManager class and exposed as a single function, for example
```objectivec
[AccountManager sharedManager].hasActiveAccounts
```
Something to consider for future development.